packages/components/src/components/Alert/Alert.tsx

```diff
@@ -194,6 +211,46 @@ export const Alert: FC<AlertProps> = ({
     lineHeight: 30,
   }
 
+  const _header = () => {
+    if (!header) return null
+
+    const headerText = <Text style={headerFont}>{header}</Text>
+    const a11yLabel = headerA11yLabel || header
+    const hitSlop: Insets = {
+      // left border + left padding + spacer + icon width
+      left: Sizing._8 + Sizing._12 + Sizing._10 + Sizing._24,
+      top: Sizing._20,
+      // bottom spacing changes depending on expanded state
+      bottom: expanded ? Sizing._10 : Sizing._20,
+      right: Sizing._20,
+    }
+
+    /**
+     * Wrap header text and expand icon in Pressable if the Alert is expandable
+     * Otherwise wrap in View with accessibility props
+     */
+    if (expandable) {
+      return (
+        <Pressable
+          onPress={() => setExpanded(!expanded)}
+          accessibilityRole="tab"
+          accessibilityState={{ expanded }}
```

Comment:
```suggestion
          aria-expanded={{ expanded }}
```
(not sure if same formatting for expanded side)